### PR TITLE
Digital subscriptions landing page: comparison table AB test

### PR DESCRIPTION
**Pull request by lucymonie**
## What are you doing in this PR?
Adding a comparison table to the digital subscriptions landing page for AB testing.

[**Trello Card**](https://trello.com/c/glrYCqiO/3771-comparison-table-ab-test-ux-100-ui-100-dev-0)
[**Figma**](https://www.figma.com/file/n2oBySn1jfa55DWb1uC8Ck/DS-LP-Q1---UI-UX-testing?node-id=2101%3A2915)

## Why are you doing this?
During qual testing, the comparison table has been well liked by readers, who say it's a helpful summary of what they get if they pay for a subscription.

## Is this an AB test?
- [x] yes
- [ ] no

The audience size in the test is currently set to zero, so this code could be merged in and the test started later by changing the audience size (and switching on the test in Optimize).

If this is an AB test, PR reviewers should open and check the Optimize test.
[**Optimize link**](https://optimize.google.com/optimize/home/#/accounts/1368065808/containers/6827913/experiments/491)

## Accessibility test checklist
 - [x] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
 - [x] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
 - [x] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

## Screenshots
Screen Shot 2021-07-13 at 10 20 27

Screen Shot 2021-07-13 at 10 20 05

### CODE REVIEW

File: support-frontend/assets/helpers/abTests/abtestDefinitions.js [hunk 1]
```diff
@@ -87,4 +87,25 @@ export const tests: Tests = {
...
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 0,
+      },
+    },
+    isActive: true,
+    referrerControlled: false,
+    targetPage: pageUrlRegexes.subscriptions.digiSub.nonGiftLandingAndCheckout,
```
Comment (lucymonie) [inline support-frontend/assets/helpers/abTests/abtestDefinitions.js:107]: This test is targeting all regions via the main digital subs landing page and checkout (as we are tracking conversions), and should not appear on the gift page or gift checkout.

**Review by i-hardy — APPROVED** [review]
Looks great!

File: support-frontend/assets/helpers/abTests/abtestDefinitions.js [hunk 2]
```diff
@@ -87,4 +87,25 @@ export const tests: Tests = {
     seed: 10,
     optimizeId: '-2DJ0JTsSVaskWwTm9Je4A',
   },
+  comparisonTableTest: {
```
Comment (rupertbates) [inline support-frontend/assets/helpers/abTests/abtestDefinitions.js:90]: Might be worth deactivating the events test as well in this PR?

File: support-frontend/assets/stylesheets/emotion/colours.js [hunk 3]
```diff
@@ -3,3 +3,5 @@ export const digitalSubscriptionsBlue = '#00568D';
 export const guardianWeeklyBlue = '#66c2e9';
 
 export const paperSubscriptionsBlue = '#90DCFF';
+
+export const comparisonTableYellow = '#FFFACC';
```
Comment (rupertbates) [inline support-frontend/assets/stylesheets/emotion/colours.js:7]: Nitpick: I'm not sure there's all that much value in defining this in a separate file as it is only used in one place, you could just move it to `tableContents.jsx`

**Review by rupertbates — APPROVED** [review]
Looks good to me, can't wait to see how this performs!

File: support-frontend/assets/stylesheets/emotion/colours.js (hunk 3 above)
Comment (lucymonie) [reply support-frontend/assets/stylesheets/emotion/colours.js]: Yes, that's fair! 👍🏻

File: support-frontend/assets/helpers/abTests/abtestDefinitions.js (hunk 2 above)
Comment (lucymonie) [reply support-frontend/assets/helpers/abTests/abtestDefinitions.js:90]: Good suggestion. I've put the audience size to zero for that test and to 1 for this one. I'll create a card to remove the events test.

**Pull request merged**